### DISCUSSION TITLE
更改“插图”为“图录”并在图录里不显示引用符号

### DISCUSSION THREAD

**Discussion by TaoYUMines** [sjtug/SJTUThesis#992] [提问] [answered]
您好！

首先想问一下如何把“插 图”改为“图 录”呢？
![微信截图_20231209211840](https://github.com/sjtug/SJTUThesis/assets/101121705/173ea3a6-a564-42e5-8c12-aaae586d9b1a)
然后，如果正文中的图片在图题加注了引用，有没有办法让图录里的图题不要显示引用，只显示标题？
整体目标效果如下图：
![微信截图_20231209211944](https://github.com/sjtug/SJTUThesis/assets/101121705/26e5163d-8381-4a33-a75b-c0190505069f)
谢谢！

**Comment by NakedRaccoon** [comment, marked as answer]
第一个问题：到模板里texmf/tex/latex/sjtutex/sjtu-scheme-zh.def文件里，把 `listfigure    = { 插 \quad 图 }` 改成`listfigure    = { 图 \quad 录 } `
第二个问题，可以用\bicaption命令，目录里的标题和图题可以分别设置
```
\begin{figure}[!htp]
  \centering
  \includegraphics[width=7cm]{figures/图片.png}
  \bicaption[目录里的图题，没有引用]
    {图题，有引用\cite{参考文献}}
    {English figure title}
  \label{标签}
\end{figure}
```

**Reply by TaoYUMines** [reply]
好的谢谢！
